### PR TITLE
Add AAD Premium dependence info

### PR DESCRIPTION
**Pull request by JeremyTBradshaw**
The issues opened with these pages show that the confusion is there and clarity is needed around this.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointOnline/sign-out-inactive-users.md [hunk 1]
```diff
@@ -26,8 +26,12 @@ description: "Learn how to set an idle session sign-out policy for SharePoint an
 This article is for global and SharePoint admins in Microsoft 365 who want to control user access to SharePoint and OneDrive data on unmanaged devices. Idle session sign-out lets you specify a time at which users are warned and subsequently signed out of Microsoft 365 after a period of browser inactivity in SharePoint and OneDrive. 
   
 > [!NOTE]
-> Idle session sign-out applies to the entire organization and can't be set for specific sites or users. 
-  
+> Idle session sign-out applies to the entire organization and can't be set for specific sites or users.  To target different settings for different users, use Azure AD Conditional Access policies instead (and see the next important note about that (AAD Premium requirement).
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sign-out-inactive-users.md:29]:
```suggestion
> Idle session sign-out applies to the entire organization and can't be set for specific sites or users. To target different settings for different users, use Azure AD Conditional Access policies instead. Also, see the next important note about AAD Premium requirement.
```

File: SharePoint/SharePointOnline/sign-out-inactive-users.md [hunk 2]
```diff
@@ -26,8 +26,12 @@ description: "Learn how to set an idle session sign-out policy for SharePoint an
...
+ 
+> [!IMPORTANT]
+> This feature relies on Azure AD Conditional Access policies being available.  You will need an AAD Premium P1 or P2 subscription for this to work.  For more information on this fact, refer to the announcment in the Azure Active Directory Identity Blog:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sign-out-inactive-users.md:32]:
```suggestion
> This feature relies on Azure AD Conditional Access policies being available. You will need an AAD Premium P1 or P2 subscription for this to work. For more information, refer to the announcment in the [Azure Active Directory Identity Blog](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/conditional-access-8220-limited-access-8221-policies-for/ba-p/245228).
```

File: SharePoint/SharePointOnline/sign-out-inactive-users.md [hunk 3]
```diff
@@ -26,8 +26,12 @@ description: "Learn how to set an idle session sign-out policy for SharePoint an
...
+> [https://techcommunity.microsoft.com/t5/azure-active-directory-identity/conditional-access-8220-limited-access-8221-policies-for/ba-p/245228](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/conditional-access-8220-limited-access-8221-policies-for/ba-p/245228)
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointOnline/sign-out-inactive-users.md:33]:
```suggestion
```

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by JeremyTBradshaw** [comment]
Thanks!

**Pull request merged**